File: LocadoraVeiculos-FrontEnd/src/app/model/Cliente.ts
import {Endereco} from './Endereco';
import { Reserva } from './Reserva';
export class Cliente{
    Id?:number = 0;
    nome:string = '';
    telefone:string = '';
    cpf:string= '';
    email:string = '';
    senha:string= '';
    endereco: Endereco = new Endereco;
    reservas: Reserva[] = []; // Lista de reservas
}

File: LocadoraVeiculos/Models/Cliente.cs
namespace LocadoraVeiculos.Models {
	public class Cliente {
		public int Id { get; set; }
		public string nome { get; set; }
		public string telefone { get; set; }
		public string email { get; set;}
		public string senha { get; set; }
		public string cpf { get; set; }
		public List<Reserva>? reservas { get; set; }
		public List<Endereco>? enderecos { get; set; }
	}
}
